com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -405,7 +409,7 @@ public void Blit(CommandBuffer cmd, RenderTargetIdentifier source, RenderTargetI
         /// <param name="sortingCriteria">Criteria to sort objects being rendered.</param>
         /// <returns></returns>
         /// <seealso cref="DrawingSettings"/>
-        public DrawingSettings CreateDrawingSettings(ShaderTagId shaderTagId, ref RenderingData renderingData, SortingCriteria sortingCriteria)
+        public static DrawingSettings CreateDrawingSettings(ShaderTagId shaderTagId, ref RenderingData renderingData, SortingCriteria sortingCriteria)
```

Comment:
This is breaking change on public API. We can make a utility static function (we have RenderingUtils static class) and but need to keep/obsolete this one.

Reply:
Reverted this change.